runelite-client/src/main/java/net/runelite/client/plugins/kourendlibrary/KourendLibraryOverlay.java

```diff
@@ -229,4 +242,15 @@ public Dimension render(Graphics2D g)
 
 		return null;
 	}
+
+	private boolean shouldHideOverlayIfDuplicateBook(@Nullable Book book)
+	{
+		ItemContainer itemContainer = client.getItemContainer(InventoryID.INVENTORY);
```

Comment:
You don't need to load the item container here anymore

Reply:
Could you elaborate? We had this discussion last time, that INVENTORY couldn't be null, but the method doesn't state specifically that -> Nullable.

Reply:
You don't use the inventory in this anymore because you load it in the Plugin. The `INVENTORY` Inventory is a special case in that it cannot be null, all other Inventories are nullable, hence why that method is marked `@Nullable`

Reply:
Ah, I looked at the wrong location, my bad.

Reply:
Removed